packages/client/src/comp/states/Base.test.tsx

```diff
@@ -69,6 +82,10 @@ const wisconsinContact: ContactData = {
 test('State Form Without Signature (Wisconsin) works', async () => {
   const history = createMemoryHistory()
 
```

Comment:
Can we aslo test if an unregistered voter?

- mock alloy call to return unregistered
- check that modal pops up via jest data-testid